docs/reference/announcements.md

```diff
@@ -10,6 +10,10 @@ Release date: 10th of October 2023
 
 End of maintenance: 9th of April 2025
 
+:::caution
+We identified some bugs in 8.3.0 that could prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
```

Comment:
Do we need to note this elsewhere in the migration documentation for now?

Reply:
from my perspective not, we have it in the migration guide and the announcements, not aware of any other relevant location 🤔

Reply:
Works for me. We can push out a release today to get this on folks' radar.